content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Dokumentacja Reacta zakłada znajomość języka programowania JavaScript na pewnym poziomie. Nie musisz być ekspertem, jednakże nauka zarówno Reacta jak i JavaScriptu w tym samym czasie może być nieco cięższa.
 
-We recommend going through [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will take you between 30 minutes and an hour but you will feel more confident learning React.
+Proponujemy zapoznanie się z [przeglądem informacji o JavaScripcie](https://developer.mozilla.org/pl/docs/Web/JavaScript/Ponowne_wprowadzenie_do_JavaScript), by sprawdzić swoją wiedzę. Powinno Ci to zająć około 30 minut do godziny, a sprawi, że nauka Reacta będzie bardziej komfortowa.
 
->Tip
+>Podpowiedź
 >
->Whenever you get confused by something in JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](http://javascript.info/) are great websites to check. There are also [community support forums](/community/support.html) where you can ask for help.
+>Gdy tylko poczujesz, że pojęcia z języka JavaScript są zbyt zagmatwane, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) oraz [javascript.info](http://javascript.info/) mogą okazać się świetnymi źródłami nauki. Istnieją także [fora internetowe](/community/support.html), na których możesz poprosić o pomoc.
 
-### Practical Tutorial {#practical-tutorial}
+### Praktyczne samouczki {#practical-tutorial}
 
-If you prefer to **learn by doing,** check out our [practical tutorial](/tutorial/tutorial.html). In this tutorial, we build a tic-tac-toe game in React. You might be tempted to skip it because you're not building games -- but give it a chance. The techniques you'll learn in the tutorial are fundamental to building *any* React apps, and mastering it will give you a much deeper understanding.
+Jeśli preferujesz **naukę przez praktykę**, sprawdź nasze [praktyczne samouczki](/tutorial/tutorial.html). W tym rozdziale stworzymy w Reakcie od podsatw grę w kółko i krzyżyk. Jeśli nie zajmujesz się tworzeniem gier i kusi Cię, by go pominąć -- najpierw daj mu szansę. Techniki, których nauczysz się w tym poradniku są fundamentami do zbudowania *dowolnej* aplikacji w Reakcie, a zrozumienie ich zapewni Ci dogłębne poznanie tematu.
 
-### Step-by-Step Guide {#step-by-step-guide}
+### Instrukcja krok po kroku {#step-by-step-guide}
 
-If you prefer to **learn concepts step by step,** our [guide to main concepts](/docs/hello-world.html) is the best place to start. Every next chapter in it builds on the knowledge introduced in the previous chapters so you won't miss anything as you go along.
+Jeśli wolisz **naukę konceptów krok po kroku**, zacznij od [przewodnika po głównych pojęciach](/docs/hello-world.html) jest najlepszym miejscem do startu. Każdy kolejny rozdział jest oparty o wiedzę nabytą w poprzednim, więc radzimy niczego nie pomijać.
```

Comment:
`konceptów` - jeżeli przy tym PR: https://github.com/reactjs/pl.reactjs.org/pull/4 wygra słowo `pojęć` lub `założeń` to tutaj trzeba będzie nanieść tę zmianę aby dokumentacja była spójna.

Reply:
Póki co zmieniam, bo konsekwentnie zmieniliśmy to w pozostałych miejsach.